README.md

```diff
@@ -2,25 +2,48 @@
 
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)
 
+## How to create a project with Sunrise data
+
+Before starting the import, make sure you have access to the [Admin Center](https://admin.commercetools.com). You will also need to run the application in this repository, check that you meet the [requirements](#requirements) and [clone](https://help.github.com/articles/cloning-a-repository/) this repository to your computer.
+
+### 1. Create your project
+1. Open the [Admin Center](https://admin.commercetools.com) and create an empty project (without sample data).
+
+### 2. Prepare node environemnt for Import
+1. Go to the root of this project, were the package.json is located.
+2. Install all node dependencies
+    ```js
+        npm install
+    ```
+3. Set commercetools project credentials as npm config values:
```

Comment:
not sure whether I did something wrong here:  when I try now I get exceptions from the java stuff that look like It lacks the necessary credentials.  Do I need to run the five config setters again every time I am in  a new terminal?

Reply:
No, but you can verify the list of configuration by `npm config list`